vit/readline.py

```diff
@@ -0,0 +1,118 @@
+import string
+import re
+
+class Readline(object):
+    def __init__(self, edit_obj):
+        self.edit_obj = edit_obj
+        word_chars = string.ascii_letters + string.digits + "_"
+        self._word_regex1 = re.compile(
+            "([%s]+)" % "|".join(re.escape(ch) for ch in word_chars)
+        )
+        self._word_regex2 = re.compile(
+            "([^%s]+)" % "|".join(re.escape(ch) for ch in word_chars)
+        )
+
+    def keys(self):
+        return ('ctrl p', 'ctrl n', 'ctrl a', 'ctrl e', 'ctrl b', 'ctrl f',
+                'ctrl h', 'delete', 'ctrl d', 'ctrl t', 'ctrl u', 'ctrl k',
+                'meta b', 'meta f', 'ctrl w', 'meta d')
+
+    def keypress(self, key):
+        if key in ('ctrl p',):
```

Comment:
I think it would help code readability to add a comment above all these key conditionals, saying what Readline action it's for, e.g.:

```python
# Move to the previous line.
if key in ('ctrl p',):
   ...
```

Reply:
Of course. I just added the descriptions. I tried to be consistent in the wording, but since I'm not a native speaker, feel free to correct any grammar mistakes!